Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -117,6 +117,68 @@
 
 #pragma mark -
 
+@interface OWSImagePickerController : UIImagePickerController
```

Comment:
I hesitate to make new source files and increase our build times.. But they're definitely clutter here.